src/cdk/schematics/BUILD.bazel

```diff
@@ -23,7 +23,11 @@ ts_library(
             "testing/**/*.ts",
         ],
     ),
+    # Schematics do not need to run in browsers and can use `commonjs`
+    # as format instead the default `umd` format.
+    devmode_module = "commonjs",
     module_name = "@angular/cdk/schematics",
+    prodmode_module = "commonjs",
```

Comment:
The schematic code does not rely on `importHelpers` / `tslib`, so it currently breaks due to the aforementioned TS bug. We can fix this by building schematics always in `commonjs`. This might generally make sense to keep since schematics do not run in the browser.